frontend/src/components/RoadmapOverview.tsx

```diff
@@ -38,35 +38,15 @@ export const RoadmapOverview = () => {
   });
 
   return (
-    <div className={classes(css.dataFlexbox)}>
-      <div className={classes(css.dataEntryWrapper)}>
-        <Trans i18nKey="Tasks" />
-        <div className={classes(css.dash)} />
-        <p className={classes(css.dataNumberWrapper)}>{roadmapTasksCount()}</p>
-      </div>
-      <div className={classes(css.dataEntryWrapper)}>
-        <Trans i18nKey="milestones" />
-        <div className={classes(css.dash)} />
-        <p className={classes(css.dataNumberWrapper)}>
-          {roadmapMilestonesCount()}
-        </p>
-      </div>
-      <div className={classes(css.dataEntryWrapper)}>
-        <Trans i18nKey="Avg Value" />
-        <div className={classes(css.dash)} />
-        <p className={classes(css.dataNumberWrapper)}>
-          {numFormat.format(value)}
-          <MonetizationOnIcon />
-        </p>
-      </div>
-      <div className={classes(css.dataEntryWrapper)}>
-        <Trans i18nKey="Avg Work" />
-        <div className={classes(css.dash)} />
-        <p className={classes(css.dataNumberWrapper)}>
-          {numFormat.format(work)}
-          <WorkIcon />
-        </p>
-      </div>
+    <div className={classes(css.data)}>
+      <MetricsSummary label="Tasks" value={roadmapTasksCount()} />
+      <MetricsSummary label="Milestones" value={roadmapMilestonesCount()} />
+      <MetricsSummary label="Avg Value" value={numFormat.format(value)}>
+        <MonetizationOnIcon />
+      </MetricsSummary>
+      <MetricsSummary label="Avg Work" value={numFormat.format(work)}>
+        <WorkIcon />
+      </MetricsSummary>
```

Comment:
I was thinking if this kind of approach would be cleaner since you wouldn't have to repeat the same component X-times and adding a new metric would be easier. Or would the metrics-array make it look messier, what do you think?

```suggestion
  const metrics = [
    {
      label: 'Tasks',
      metricsValue: roadmapTasksCount(),
    },
    {
      label: 'Milestones',
      metricsValue: roadmapMilestonesCount(),
    },
    {
      label: 'Avg Value',
      metricsValue: numFormat.format(value),
      children: <MonetizationOnIcon />,
    },
    {
      label: 'Avg Work',
      metricsValue: numFormat.format(work),
      children: <WorkIcon />,
    },
  ];

  return (
    <div className={classes(css.data)}>
      {metrics.map(({ label, metricsValue, children }) => (
        <MetricsSummary key={label} label={label} value={metricsValue}>
          {children}
        </MetricsSummary>
      ))}
    </div>
  );
```